hbase-server/src/main/java/org/apache/hadoop/hbase/snapshot/SnapshotDescriptionUtils.java

```diff
@@ -432,6 +430,40 @@ public static void completeSnapshot(Path snapshotDir, Path workingDir, FileSyste
     }
   }
 
+  static boolean shouldSkipRenameSnapshotDirectories(URI workingURI, URI rootURI) {
+    // check scheme, e.g. file, hdfs
+    if (
+      workingURI.getScheme() == null
+        && (rootURI.getScheme() != null && !rootURI.getScheme().equalsIgnoreCase("file"))
```

Comment:
Why we need the extra check for the "file" URI? Wouldn't it be enough to return "true" here if workingURI scheme is null and rootURI is not?

Reply:
with "file", we could have ignored local fs, but you are right that we don't need such explicit check, let me make this change, thanks